picosystem_macros/src/atlas.rs

```diff
@@ -88,11 +94,8 @@ pub fn atlas(input: TokenStream) -> TokenStream {
                 r#"
         pub fn {}{}() -> &'static picosystem::tile::Tile {{
             static COMPRESSION_RATIO: u32 = {};
-            #[link_section = ".static_rodata"]
```

Comment:
The separate section is helpful because it makes flashing faster when used with "pictotool -u". Did it cause problems?

Reply:
Yes, the sprite sheets were not reliably included. I used cargo-run to flash the binary directly and the elf2uf2-rs tool to create a uf2 file.